installation/scripts/prow/kyma-integration-on-debian/deploy-and-test-kyma.sh

```diff
@@ -13,5 +13,4 @@ sudo ${INSTALLATION_DIR}/cmd/run.sh --vm-driver "none"
 sudo ${INSTALLATION_DIR}/scripts/is-installed.sh --timeout 30m
 ELAPSED_TIME=$(($SECONDS - $START_TIME))
 echo "Installation of Kyma took $(($ELAPSED_TIME / 60)) minutes and $(($ELAPSED_TIME % 60)) seconds."
-sudo ${INSTALLATION_DIR}/scripts/watch-pods.sh
```

Comment:
Saw this script is used here:
https://github.com/kyma-project/test-infra/blob/36739d3856969338895573459e31810b7ad6371c/prow/scripts/minikube-test.sh#L103

Did you run tests that use `minikube-test.sh` and are they fine?